src/main/java/gov/nih/ncats/molwitch/inchi/Inchi.java

```diff
@@ -163,13 +163,13 @@ public static Chemical toChemical(String inchi) throws IOException{
 		}
 		Throwable throwable = null;
 		for(InchiImplFactory impl : implLoaders.get()) {
-
 			try {
 				Chemical result = impl.parseInchi(input);
 				if (result != null) {
 					return result;
 				}
 			}catch(Throwable t){
+				System.err.printf("Error running impl: %s\n", t.getMessage());
```

Comment:
remove debug

Reply:
I'd like to leave this item in place.
It's not debug.  This line is regularly generating errors on one system.
Will provide details offline